### PR TITLE
[TIMOB-19029] Report errors on JS parse error

### PR DESCRIPTION
**Pull request by infosia**
Fix for [TIMOB-19029](https://jira.appcelerator.org/browse/TIMOB-19029).

### CODE REVIEW

File: cli/commands/_build.js [hunk 1]
```diff
@@ -1417,8 +1447,15 @@ WindowsBuilder.prototype.copyResources = function copyResources(next) {
...
 
 				// Look for native requires here
-				// FIXME Avoid parsing the AST twice! We do it below using jsanalyze for non html referenced JS files!
-				var toplevel = UglifyJS.parse(fs.readFileSync(from).toString());
+				var fromContent = fs.readFileSync(from, {encoding:'utf8'});
+				try {
+					// FIXME Avoid parsing the AST twice! We do it below using jsanalyze for non html referenced JS files!
+					var toplevel = UglifyJS.parse(fromContent);
+				} catch  (E) {
+					t_.logger.error(reportJSErrors(from, fromContent, E));
```
Comment (dawsontoth) [inline cli/commands/_build.js]: Should this be fatal (and kill the build)? Or, should it kill the build at some point? When I run this, I see the error message, but then the emulator starts and the CLI just sits there until I terminate it.

Comment (sgtcoolguy) [inline cli/commands/_build.js]: I don't _think_ it should be fatal. The parsing here is done to sniff for native requires. If we can't parse the JS for some reason that's not a great sign, but I think we can try and move on and assume no native APIs were used in the file with bad syntax.

Comment (dawsontoth) [inline cli/commands/_build.js]: OK then.

**Comment by dawsontoth** [comment]
**CR** and **FR** passed.

**Pull request merged**